website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-snowflake-sso.md

```diff
@@ -67,3 +67,12 @@ Enter the Client ID and Client Secret into dbt Cloud to complete the creation of
 ### Authorize Developer Credentials
 
 Once Snowflake SSO is enabled, users on the project will be able to configure their credentials in their Profiles. By clicking the "Connect to Snowflake Account" button, users will be redirected to Snowflake to authorize with the configured SSO provider, then back to dbt Cloud to complete the setup process. At this point, users should now be able to use the dbt IDE with their development credentials.
+
+### SSO OAuth Flow Diagram
+
+![image](https://user-images.githubusercontent.com/46451573/84427818-841b3680-abf3-11ea-8faf-693d4a39cffb.png)
```

Comment:
I think this is not important for this PR, as it sounds like these might get updated soon, but when adding images it's good to:
1. Add the image to the repo (in the [`img` directory](https://github.com/fishtown-analytics/docs.getdbt.com/tree/master/website/static/img)) rather thank linking to a hosted image. If so far you've been using the GitHub interface to make changes to the docs, now might be the time to clone the repo to make this process easier 🙂 . I made [this Loom](https://www.loom.com/share/7037780b86eb4f16953664b8f15f1e21) to talk Alexandra through how to get the repo set up.
2. Use the `lightbox` component ([example](https://github.com/fishtown-analytics/docs.getdbt.com/blame/master/website/docs/tutorial/1-setting-up.md#L88)).